msbuild/Xamarin.MacDev.Tasks.Core/Sdks.cs

```diff
@@ -65,5 +67,30 @@ public static AppleSdk GetSdk (string targetFrameworkMoniker)
 		{
 			return GetSdk (PlatformFrameworkHelper.GetFramework (targetFrameworkMoniker));
 		}
+
+		public static IAppleSdk GetAppleSdk (ApplePlatform framework)
+		{
+			switch (framework) {
+			case ApplePlatform.iOS:
+				return IOS;
+			case ApplePlatform.WatchOS:
+				return Watch;
+			case ApplePlatform.TVOS:
+				return TVOS;
+			case ApplePlatform.MacOSX:
+				return MacOS;
+			case ApplePlatform.MacCatalyst:
+				return IOS;
+			default:
+				throw new InvalidOperationException (string.Format ("Invalid framework: {0}", framework));
```

Comment:
not a new issue but... should not this be translated like other messages ?

Reply:
probably